src/org/rascalmpl/repl/TerminalProgressBarMonitor.java

```diff
@@ -439,14 +439,6 @@ static String moveUp(int n) {
             return "\u001B[" + n + "F";
         }
 
-        static String overlined() {
```

Comment:
this is a unexpected change in this PR?

Reply:
I want to see I did not produce new warnings. These were unused. I really would like to have "warnings as errors" on on this project, but that seems a bit far from realistic due to the warnings that the rascal compiler produces.